client/src/utils/secureAuth.js

```
@@ -0,0 +1,84 @@
+					.map(c => '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2))
+					.join('')
+			);
+			return JSON.parse(jsonPayload);
+		} catch (error) {
+			return null;
+		}
+	}
+
+	// Check if user is authenticated
+	static isAuthenticated() {
+		return this.getToken() !== null && this.getUserData() !== null;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)